plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/JdbcTableHandle.java

```diff
@@ -146,6 +146,16 @@ public JdbcNamedRelationHandle asPlainTable()
         return getRequiredNamedRelation();
     }
 
+    public JdbcTableHandle toPlainTableWithoutColumns()
```

Comment:
I would like not to have this method, here and move it to the place where it is needed.